modules/smarthome/elwa/watt.py

```diff
@@ -40,7 +40,8 @@
 # Wassertemperatur lesen
 value1 = resp.registers[1]
 all = format(value1, '04x')
-temp0 = int(struct.unpack('>h', codecs.decode(all, 'hex'))[0])
+temp0int = int(struct.unpack('>h', codecs.decode(all, 'hex'))[0])
+temp0 = float(temp0int) / 10
```

Comment:
```suggestion
temp0 = temp0int / 10
```
Den Typecast braucht es hier wegen des Duck-Typings in Python nicht.